index.html

```diff
@@ -0,0 +1,206 @@
+<html>
+    <head>
+        <title>YouTube</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body> 
+
+<nav class="flex-div">
+    <div class="nav-left flex-div">    
+    <img src="images/menu.png" class="menu-icon">
+    <img src="images/logo.png" class="logo">
+        </div>
+        <div class="nav-middle flex-div">
+            <div class="search-box flex-div">
+
+                <input type="search" placeholder="search">
+                <img src="images/search.png" class="search-icon">
+            </div>
+            <img src="images/voice-search.png" class="mic-icon">
+        </div>
+        <div class="nav-right flex-div">
+            <img src="images/upload.png">
+            <img src="images/more.png">
+            <img src="images/notification.png">
+            <img src="images/Jack.png" class="user-icon">
+            
+        </div>
+    </nav>
+
+
+    <!-- -----------side-bar------------- -->
+
+    <div class="side-bar">
+        <div class="shortcut-links">
+            <a href="#"><img src="images/home.png"><p>Home</p></a>
+            <a href="#"><img src="images/explore.png"><p>Explore</p></a>
+            <a href="#"><img src="images/subscriprion.png"><p>Subscription</p></a>
+            <a href="#"><img src="images/library.png"><p>Library</p></a>
+            <a href="#"><img src="images/history.png"><p>History</p></a>
+            <a href="#"><img src="images/playlist.png"><p>Playlist</p></a>
+            <a href="#"><img src="images/messages.png"><p>Messages</p></a>
+            <a href="#"><img src="images/show-more.png"><p>Show More</p></a>
+            <hr>
+        </div>
+            <div class="subscribed-list">
+                <h3>SUBSCRIBED</h3>
+                <a href="#"><img src="images/Jack.png"><p>Jack Man</p></a>
+                <a href="#"><img src="images/megan.png"><p>Megan Foxy</p></a>
+                <a href="#"><img src="images/cameron.png"><p>Cameron kalley</p></a>
+                <a href="#"><img src="images/tom.png"><p>Tom Croissant</p></a>
+                <a href="#"><img src="images/simon.png"><p>Simon Showmen</p></a>
+            </div>
+        
+    </div>
+<!-- -------- Main--------- -->
+<div class="container">
```

Comment:
- [ ] [OPTIONAL] This comment is nearly perfect, it's polite, clear and descriptive, however, I believe it could be further improved by referring to the official project requirements either by using a link to the[ requirements page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) or by quoting the requirement specific to this case like this 🔽 
> Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.